torch/_dynamo/optimizations/torchxla_integration.py

```diff
@@ -39,7 +38,9 @@ def __call__(self, args):
             self.graph_input_tensor_ids, self.graph_input_xla_values
         ):
             arg_idx = self.tensor_id_to_arg_idx.get(tensor_id, None)
-            if arg_idx is None:
+            if tensor_id == torch_xla._XLAC._get_seed_info_id():
```

Comment:
maybe add a comment here explain why we need to extract the random seed (to prevent same rng being used by different steps) here?